IBuildaAZoo/IBuildaAZoo/Elephant.cs

```diff
@@ -3,8 +3,12 @@ namespace IBuildaAZoo
 {
     public class Elephant
     {
-        public Elephant()
+        public abstract class Elephant()
         {
-        }
+        public abstract int NumberOfLegs = 4;
+        public string Size = "Extra Large"
+        public virtual string Skin = "No Fur";
```

Comment:
This would be an `override` to replace `"Fur"` default from `Animal`.
```suggestion
        public override string Skin => "No Fur";
```